package.json

```diff
@@ -43,37 +43,37 @@
   },
   "devDependencies": {
     "@cdk8s/projen-common": "0.0.530",
-    "@cdktf/provider-aws": "^17.0.11",
+    "@cdktf/provider-aws": "^19.0.0",
     "@types/fs-extra": "^11.0.4",
-    "@types/jest": "^27",
+    "@types/jest": "^29",
     "@types/node": "16.18.78",
     "@typescript-eslint/eslint-plugin": "^7",
     "@typescript-eslint/parser": "^7",
     "cdk8s": "^2.68.4",
     "cdk8s-cli": "^2.198.154",
-    "cdktf": "^0.19.1",
+    "cdktf": "^0.20.7",
     "cdktf-cli": "^0.20.7",
     "constructs": "^10.3.0",
     "eslint": "^8",
     "eslint-import-resolver-typescript": "^3.6.1",
     "eslint-plugin-import": "^2.29.1",
     "fs-extra": "^11.2.0",
-    "jest": "^27",
```

Comment:
How come?

Reply:
Jest needs to be updated to fix the warning that it has not been tested for typescript 5. Also the conversation in the fix notes indicates that even though some later versions said it worked, it didn't.

https://github.com/kulshekhar/ts-jest/issues/4048